### DISCUSSION TITLE
Rencontre du 26 juin

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#33] [Rencontres d'équipe]
Bonjour à vous deux, je vous propose qu'on se rencontre aujourd'hui à 10h30 pour discuter des points suivants.

1. Horaire plus officiel des srums matinaux
2. Communication par rapport à notre présence ou pas en classe par rapport au Srum et au travail d'équipe en général.
3. Ajouter vos points au besoin.

Êtes-vous disponible aujourd'hui, mercredi à 10h30 @claraquintela et @NabilDjerroud ?

**Comment by claraquintela** [comment]
Parfait!
J'y serais :)

**Reply by NabilDjerroud** [reply]
Moi aussi j'y serais 
À tantôt!

**Comment by claraquintela** [comment]
Hangout?

Voici le lien:

https://meet.google.com/efb-bupq-hdp

La salle sera ouverte 10h30



Em qua., 26 de jun. de 2024 às 09:36, Nabil ***@***.***>
escreveu:
